compiler/noirc_frontend/src/hir/comptime/errors.rs

```diff
@@ -394,8 +394,7 @@ impl<'a> From<&'a InterpreterError> for CustomDiagnostic {
                 let mut diagnostic =
                     CustomDiagnostic::simple_error(primary, secondary, location.span);
 
-                // Only take at most 3 frames starting from the top of the stack to avoid producing too much output
-                for frame in call_stack.iter().rev().take(3) {
+                for frame in call_stack.iter().rev() {
```

Comment:
Why was the `.take(3)` limit removed?

Reply:
Oh, I forgot to mention this. With just 3 sometimes I couldn't understand the main source of the error. I changed it to 5 first and that worked, but I feel that it depends on each case. Maybe for now we can leave it unlimited until we find it's a issue.

Reply:
I think we should still have some bound on it. I don't want cases where the user is 20+ functions deep and getting such an error. Maybe just bump it up to 5 for now?